src/elkar/server_mcp.py

```diff
@@ -0,0 +1,385 @@
+    """
+    # If agent_urls not provided, use environment variable
+    if agent_urls is None:
+        # Get from environment and split by comma
+        env_urls = os.environ.get('AGENT_URLS', 'http://localhost:5001')
+        agent_urls = [url.strip() for url in env_urls.split(',')]
+    
+    results = {
+        "available_agents": [],
+        "errors": []
+    }
+    
+    for url in agent_urls:
+        try:
+            # Configure the A2A client for this URL
+            config = A2AClientConfig(
+                base_url=url,
+                headers=headers,
+                timeout=timeout
+            )
+            
+            # Use the client as a context manager for proper resource management
+            async with A2AClient(config) as client:
+                try:
+                    # Retrieve the agent card
+                    agent_card = await client.get_agent_card()
+                    
+                    # Extract key information from the agent card
+                    agent_info = {
+                        "url": url,
+                        "name": agent_card.name,
+                        "description": agent_card.description,
+                        "version": agent_card.version,
+                        "capabilities": {
+                            "streaming": agent_card.capabilities.streaming if hasattr(agent_card.capabilities, "streaming") else False,
+                            "pushNotifications": agent_card.capabilities.pushNotifications if hasattr(agent_card.capabilities, "pushNotifications") else False,
+                            "stateTransitionHistory": agent_card.capabilities.stateTransitionHistory if hasattr(agent_card.capabilities, "stateTransitionHistory") else False,
+                        }
+                    }
+                    
+                    # Include skills information if available
+                    if hasattr(agent_card, "skills") and agent_card.skills:
+                        agent_info["skills"] = [
+                            {"id": skill.id, "name": skill.name} 
+                            for skill in agent_card.skills
+                        ]
+                    
+                    results["available_agents"].append(agent_info)
+                    
+                except Exception as e:
+                    # Add to errors if we can connect but can't get the agent card
+                    results["errors"].append({
+                        "url": url,
+                        "error": f"Error retrieving agent card: {str(e)}"
+                    })
+                    
+        except Exception as e:
+            # Add to errors if we can't even connect to the URL
+            results["errors"].append({
+                "url": url,
+                "error": f"Failed to connect to A2A agent: {str(e)}"
+            })
+    
+    return results
+
+if __name__ == "__main__":
+    mcp.run("stdio")
```

Comment:
create CLI for MCP 

should be run with `uvx` 

Take example here https://github.com/e2b-dev/mcp-server/blob/main/packages/python/README.md

create mode with sse and stdio